src/promptflow/promptflow/_cli/_configuration.py

```diff
@@ -0,0 +1,57 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import json
+import os.path
+import uuid
+from pathlib import Path
```

Comment:
I think you already check in this, can we refresh the PR to only show the new changes?

Reply:
Removed